src/utils/endUserMessages.js

```diff
@@ -18,6 +18,7 @@ export default {
   ERROR_FETCHING_BACKEND_STATUS: 'We couldn\'t get the status of processing the experiment\'s processing run.',
   ERROR_NOT_SIGNED_IN: 'You need to sign in first. Click Sign in on the top right to begin.',
   ERROR_SAVING: 'We couldn\'t save your data.',
+  ERROR_CREATING_SAMPLE: 'We couldn\'t save your data. Please reupload sample:',
```

Comment:
Isn't it more correct to say "We couldn't create your sample. Please try uploading it again"?

Reply:
Yes, I think it's more accurate. I'll change it